ReactQt/runtime/src/qml/ReactPicker.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import React 0.1 as React
+import QtQuick.Extras 1.4
+import QtQuick.Controls.Styles 1.2
+
+
+Tumbler {
+    id: tumbler
+    property var pickerManager: null
+    property variant flexbox: React.Flexbox {control: tumbler}
+    property variant p_items: []
+    property int p_selected:0
```

Comment:
<Picker> component in React Native doesn't have `selected` property, it is called `selectedValue`. I would suggest to remove `p_` prefix because it is used to match original React Native properties by name.
Also it would be nice to add `p_selectedValue` property because without it we can't tie `Picker` value to another variable in code.